packages/ui-components/src/table/render-row.tsx

```diff
@@ -76,7 +79,7 @@ const Cell = styled.div`
     align-items: center;
 
     min-width: 80px;
-    height: 40px;
+    height: 2.5rem;
```

Comment:
Shouldn't this also be `ROW_HEIGHT`?

Reply:
Yeah, they should always be the same because `ROW_HEIGHT` will just be the same value in pixels due to the way it is computed, but I've changed it now to follow `ROW_HEIGHT` so that we won't need to change two places if we change the computations. Good catch.